src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculator {
```

Comment:
The name of the interface `AreaCalculator` implies an action (calculate), which is not recommended for class/interface names. Consider renaming the interface to something that represents an entity or a capability like `AreaComputable` or `AreaProvider`.